anime-list-master.xml

```diff
@@ -12543,6 +12543,9 @@
   </anime>
   <anime anidbid="4243" tvdbid="79839" defaulttvdbseason="2" episodeoffset="" tmdbid="" imdbid="">
     <name>Aria the Natural</name>
+    <mapping-list>
+      <mapping anidbseason="0" tvdbseason="0">;204-1;</mapping>
```

Comment:
What's this about episode 204? I know that you want to map T04 to TVDB but I haven't seen anything in anime-lists's documentation about it.

Reply:
It works on my end using HAMA. Here's a post I found from ScudLee regarding mapping Ts: https://github.com/ScudLee/anime-lists/issues/150